Makefile

```diff
@@ -14,6 +15,9 @@ deps:
 build: clean deps
 	$(COMMONENVVAR) $(BUILDENVVAR) godep go build -o kube-state-metrics 
 
+crossbuild:
```

Comment:
I don't think we will start releasing in any other form than the container images for now. At least until google opens up the release process further. So for now this will just be a helper to build it yourself.